elements/campaign-display/components/campaign-display-image.js

```diff
@@ -3,13 +3,14 @@ import React, { PropTypes } from 'react';
 export default function CampaignDisplayImage (props) {
   return (
     <div className='campaign-display-image'>
-      <a href={props.url}>
-        <img src={props.image} />
+      <a href={props.clickthrough_url}>
```

Comment:
I'd like to wait until there are actual requirements to inform this decision. I have no knowledge of any other attribute needs.